### PR TITLE
READY : Extend documentation

### PR DESCRIPTION
**Pull request by dmvict**

### CODE REVIEW

**Comment by Wandalen** [comment]
Redundant renaming

**Comment by dmvict** [comment]
> Redundant renaming

Replaced. 
The ESlint errors : 
```
...
  19309:22  error  Expected error to be handled  handle-callback-err
  19350:22  error  Expected error to be handled  handle-callback-err
  20099:22  error  Expected error to be handled  handle-callback-err
  20141:22  error  Expected error to be handled  handle-callback-err
  20182:22  error  Expected error to be handled  handle-callback-err

✖ 102 problems (102 errors, 0 warnings)
```

File: proto/wtools/abase/l9.test/consequence/Consequence.test.s [hunk 1]
```diff
@@ -183,7 +183,7 @@ function fromAsyncMode00( test )
   .then( function( arg )
   {
     test.case = 'passing rejected promise';
-    var src = Promise.reject( 'str' );
+    var src = Promise.reject( _.err( 'str' ) );
```
Comment (Wandalen) [inline proto/wtools/abase/l9.test/consequence/Consequence.test.s:186]: ?

Comment (dmvict) [reply proto/wtools/abase/l9.test/consequence/Consequence.test.s:188]:
```
  wConsequence/proto/wtools/abase/l9.test/consequence/Consequence.test.s
  5924:15  error  Expected the Promise rejection reason to be an Error  prefer-promise-reject-errors
``` 

The thrown or rejected values should be an Error.

Comment (dmvict) [reply proto/wtools/abase/l9.test/consequence/Consequence.test.s]: Removed extra wrapping.

**Pull request merged**